lib/config-schema.ts

```diff
@@ -184,6 +184,7 @@ const ConfigSchema = z
         z
           .object({
             accountId: AWSAccountId,
+            accountNameRegex: z.string().optional(),
```

Comment:
Minor idea, could also add regex to the exclude logic by either specifying accountId or accountRegex in excludeList.

Reply:
Could do. No need to add it until required.

Reply:
Could do. No need to add it until required.